File: testdata/fixtures/nodejs-service/src/otel.ts
import { trace } from '@opentelemetry/api';

const tracer = trace.getTracer('node-checkout');
const span = tracer.startSpan('reconcile-orders');
span.setAttribute('user.id', '42');
span.end();

export const resource = {
  'service.name': 'node-checkout'
};

File: testdata/fixtures/python-service/src/otel.py
from opentelemetry import trace
from opentelemetry.sdk.resources import Resource

resource = Resource.create({"service.name": "python-checkout"})
tracer = trace.get_tracer("checkout")

with tracer.start_as_current_span("reconcile-orders") as span:
    span.set_attribute("user.id", "42")
